crates/bevy_pbr/src/atmosphere/aerial_view_lut.wgsl

```diff
@@ -46,14 +47,25 @@ fn main(@builtin(global_invocation_id) idx: vec3<u32>) {
             let local_up = get_local_up(r, t_i, ray_dir.xyz);
 
             let local_atmosphere = sample_atmosphere(local_r);
-            optical_depth += local_atmosphere.extinction * dt; 
+            let sample_optical_depth = local_atmosphere.extinction * dt;
+            let sample_transmittance = exp(-sample_optical_depth);
+            optical_depth += sample_optical_depth;
 
             // use beer's law to get transmittance from optical density
             let transmittance_to_sample = exp(-optical_depth);
 
             // evaluate one segment of the integral
-            var local_inscattering = sample_local_inscattering(local_atmosphere, transmittance_to_sample, ray_dir.xyz, local_r, local_up);
-            total_inscattering += local_inscattering * dt;
+            var inscattering = sample_local_inscattering(local_atmosphere, vec3(1.0), ray_dir.xyz, local_r, local_up);
```

Comment:
We set the `transmittance_to_sample` parameter of the `sample_local_inscattering` function to `vec3(1.0)` in case of using analytical integration.